libsolutil/UTF8.h

```diff
@@ -38,4 +38,6 @@ inline bool validateUTF8(std::string const& _input)
 	return validateUTF8(_input, invalidPos);
 }
 
+size_t offsetToUTFCodepoint(std::string const& _input, size_t _offset);
```

Comment:
Are you sure you mean `codepoint` here? What does this function return?

Reply:
Ah, the name is ambigious I guess. It returns the amount of codepoints at that byte offset

Reply:
`numBytesToCodepoints`?

Reply:
Can you make it take a string without offset?

Reply:
`size_t numCodepoints(std::string const& _utf8EncodedInput)`?

Reply:
By the way, please check what happens if we start or end the string inside a multi-byte character. I think we do not have to handle that nicely, but it should also not fail.

Reply:
The function only counts the beginning of codepoints, so if we start in the middle, we get 0 and if we end early we still get one.